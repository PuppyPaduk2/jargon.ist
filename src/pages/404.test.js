// @flow

import React from 'react';
import { shallow } from 'enzyme';

import NotFound from './404';

it('renders correctly', () => {
  const wrapper = shallow(<NotFound />);

  expect(wrapper).toMatchSnapshot();
});
