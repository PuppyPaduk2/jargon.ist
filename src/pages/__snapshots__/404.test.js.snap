// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Section
  className=""
>
  <Container
    className=""
  >
    <h1>
      Sayfa bulunamadı
    </h1>
    <p>
      Aradığınız sayfa bulunamadı. Eğer aradıgınız şey bir jargon ise, bize
       
      <a
        href="https://github.com/alpcanaydin/jargon.ist"
      >
        katkıda bulunarak
      </a>
       jargon.ist'e ekleyebilirsin.
    </p>
  </Container>
</Section>
`;
